lib/get_started/get_started_view.dart

```diff
@@ -0,0 +1,111 @@
+      backgroundColor: const Color(0xFFf2e9e0),
+      body: CustomPaint(
+        size: const Size(double.infinity, double.infinity),
+        painter: CustomBackgroundPainter(),
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 8.0),
+          child: Stack(
+            children: [
+              Positioned(top: 0,right: -24,
+                  child: Image.asset('assets/images/tarelka-removebg-preview.png',height: 300,width: 300,fit: BoxFit.cover,)),
+              Positioned(top: 160,left: 70,
+                  child: Image.asset('assets/images/mint_leaf-removebg-preview.png',height: 64,width: 64,)),
+              Positioned(top: 230,left: 140,child: Image.asset('assets/images/mint_leaf-removebg-preview.png',height: 50,width: 50,)),
+              Positioned(top: 235,left: 80,child: Image.asset('assets/images/strawberry2-removebg-preview.png',height: 50,width: 50,)),
+              Column(
+                children: [
+                  const Spacer(flex: 4),
+                  const Align(
+                    alignment: Alignment.topLeft,
+                    child: Padding(
+                      padding: EdgeInsets.only(left: 28.0),
+                      child: Text(
+                        'Lorem ipsum dolor sit amet,\nconsectetur adipiscing elit sed.',
+                        style: TextStyle(
+                            fontSize: 18,
+                            color: Colors.brown,
+                            fontWeight: FontWeight.bold),
+                      ),
+                    ),
+                  ),
+                  const SizedBox(height: 20),
+                  const Align(
+                    alignment: Alignment.topLeft,
+                    child: Padding(
+                      padding: EdgeInsets.only(left: 28.0),
+                      child: Text(
+                        'Tempor incididunt ut labore et \ndolore magna aliqua consectetur \nadipiscing elit sed.',
+                        style: TextStyle(
+                          fontSize: 18,
+                          color: Colors.brown,
+                        ),
+                      ),
+                    ),
+                  ),
+                  const Spacer(flex: 2),
+                  Align(
+                    alignment: Alignment.center,
+                    child: ElevatedButton(
+                      onPressed: () {},
+                      style: ElevatedButton.styleFrom(
+                        padding: const EdgeInsets.symmetric(vertical: 16.0,horizontal: 32.0),
+                        foregroundColor: const Color(0xFFf3e1d4),
+                        backgroundColor: const Color(0xFFf3e1d4),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(30),
+                        ),
+                      ),
+                      child: const Text('Get Started',style: TextStyle(color: Color(0xFFaa9184),fontSize: 18),),
```

Comment:
![image](https://github.com/user-attachments/assets/a5514104-b092-4a30-aa11-5b4b41f8089e)
Font style is different

Reply:
font style updated